cla_frontend/assets-src/javascripts/app/partials/case_detail.outcome.html

```diff
@@ -33,10 +33,9 @@
         <span
           class="CaseHistory-logItemNotes"
           ng-if="log.notes && log.code !== 'MT_CREATED' && log.code !== 'MT_CHANGED'"
-          ng-text-truncate="log.notes"
-          ng-tt-words-threshold="10"
-          ng-tt-more-label="Show more"
-          ng-tt-less-label="Show less"></span>
+        >
+          {{ ::log.notes }}
```

Comment:
This is going to change the users experience if we merge this change. I see that you have a second PR opened that handles sanitising the notes, I assume there will be a further PR that handles how we show the user the truncated sanitized text and have `more`/`less` labels available to the user (which i am assuming expands/contracts the whole text string.)